Sources/Toast/ToastViewModifier.swift

```diff
@@ -38,13 +39,15 @@ public extension View {
         style: Warp.ToastStyle,
         title: String,
         edge: Warp.ToastEdge,
+        horizontalPadding: Double = 0,
```

Comment:
I don't see any requirements from Warp to be able to set the `horizontalPadding`. Is that something your designers asked for? So I can report back to the team that this is something that our users need so we can document it and other platforms can add it too. And should the default value be 16 so by default it has a padding? or 0 is okey?

Reply:
I have asked our designers for feedback regarding padding rules on this component

Reply:
I have this design.

![Toast_example](https://github.com/warp-ds/warp-ios/assets/117636/ccc20abe-6fd2-4cf6-bb35-ff55aa51cb51)

But maybe we should change the default to 16 and lock it at that...just so your toasts don't go wall to wall but you can't tweak them yourself?

Reply:
We have a response from our designers. It should be always constant and 16 from the sides.

Reply:
Cool, I've updated already and locked it so it can't be modified. I'll merge